src/Func.h

```diff
@@ -853,6 +853,12 @@ class Func {
      * splitting it up. If bounds inference decides that it requires
      * more of this function than the bounds you have stated, a
      * runtime error will occur when you try to run your pipeline.
+     *
+     * The min argument may be an undefined Expr, in which case the
```

Comment:
That had occurred to me :) It's more verbose than I'd like. I couldn't decide between a bound overload and bound_extent, or if this should exist at all.